prospector/rules/helpers.py

```diff
@@ -197,25 +198,37 @@ def extract_path_similarities(commit: Commit, advisory_record: AdvisoryRecord):
     return similarities
 
 
-def fetch_candidate_references(commit: Commit) -> Commit:
-    # FIXME: this is very ad-hoc for GH issue/PR pages
-
-    for ref, page_content in commit.ghissue_refs.items():
-        if page_content is None:
-            url = commit.repository + "/issues/" + ref.lstrip("#")
-            raw_page_content = fetch_url(url)
-
-            soup = BeautifulSoup(raw_page_content, "html.parser")
-            content = ""
-            for comment_block in soup.find_all(
-                True, class_=["markdown-body", "markdown-title"]
-            ):
-                content += comment_block.text
-
-            if len(content) > 0:
-                commit.ghissue_refs[ref] = content
-
-    # TODO: also treat JIRA pages
-    # TODO: cache BS extracted text (it takes some time...)
-
-    return commit
+# def fetch_candidate_references(commit: Commit) -> Commit:
```

Comment:
Is this commented out because it is implemented somewhere else?

If you think this should be kept commented out (for now), please add a (meta-) comment at the top to explain